versioning/BRM/extentmap.cpp

```diff
@@ -219,15 +219,20 @@ boost::mutex ExtentMapRBTreeImpl::fInstanceMutex;
 ExtentMapRBTreeImpl* ExtentMapRBTreeImpl::fInstance = nullptr;
 
 /*static*/
-ExtentMapRBTreeImpl* ExtentMapRBTreeImpl::makeExtentMapRBTreeImpl(unsigned key, off_t size, bool readOnly)
+ExtentMapRBTreeImpl* ExtentMapRBTreeImpl::makeExtentMapRBTreeImpl(unsigned key, off_t size, bool readOnly,
+                                                                  const MasterSegmentTable* emSegTable)
```

Comment:
`emSegTable` should not be const here.

Reply:
Why?

Reply:
It is a constant pointer to a MST and I don't change it so I don't see any reason to remove const here.

Reply:
I just noticed `getTable_upgrade` and `getTable_downgrade` are const functions, so this looks good to me.